### PR TITLE
pd: add some configuration descriptions

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### What is changed, added or deleted? (Required)

- add a description of `initial-cluster-token`
- add some other missing descriptions

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4183
- Other reference link(s): https://github.com/pingcap/pd/pull/2691

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: pd-configuration-file.md [hunk 1]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
+
+- The listening URL list for peer traffic
+- Default value: `"http://127.0.0.1:2380"`
+- To deploy a cluster, you must use `peer-urls` to specify the IP address of the current host, such as `"http://192.168.100.113:2380"`. If the cluster runs on Docker, specify the IP address of Docker as `"http://0.0.0.0:2380"`.
+
+### `advertise-peer-urls`
+
+- The advertise URL list for peer traffic from outside
+- Default: `"${peer-urls}"`
+- If the peer cannot connect to PD through the default listening peer URLs, you must manually set the advertise peer URLs explicitly.
```
Comment (TomShawn) [inline pd-configuration-file.md:49]:
```suggestion
- In some situations such as the Docker or NAT network environment, if the other nodes cannot connect to PD through the default peer URLs to be listened to by PD, you must manually set the advertise peer URLs.
```

File: pd-configuration-file.md [hunk 2]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
+- For example, the internal IP address of Docker is 172.17.0.1, while the IP address of the host is 192.168.100.113 and the port mapping is set to `-p 2380:2380`. In this case, you can set `advertise-peer-urls` to `"http://192.168.100.113:2380"`. The other PD nodes can find this service through `"http://192.168.100.113:2380"`.
```
Comment (TomShawn) [inline pd-configuration-file.md:50]:
```suggestion
- For example, the internal IP address of Docker is `172.17.0.1`, while the IP address of the host is `192.168.100.113` and the port mapping is set to `-p 2380:2380`. In this case, you can set `advertise-peer-urls` to `"http://192.168.100.113:2380"`. The other PD nodes can find this service through `"http://192.168.100.113:2380"`.
```

File: pd-configuration-file.md [hunk 3]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
+    ```
+
+### `initial-cluster-state`
+
++ The initial state of the cluster
++ Default value: `"new"`
+
+### `initial-cluster-token`
+
++ Used to identify different clusters during the bootstrap phase.
```
Comment (TomShawn) [inline pd-configuration-file.md:70]:
```suggestion
+ Identifies different clusters during the bootstrap phase.
```

File: pd-configuration-file.md [hunk 4]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
++ Default value: "pd-cluster"
```
Comment (TomShawn) [inline pd-configuration-file.md:71]:
```suggestion
+ Default value: `"pd-cluster"`
```

File: pd-configuration-file.md [hunk 5]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
++ If multiple clusters that have nodes with same configurations are deployed successively, different tokens should be specified to isolate different cluster nodes.
```
Comment (TomShawn) [inline pd-configuration-file.md:72]:
```suggestion
+ If multiple clusters that have nodes with same configurations are deployed successively, you must specify different tokens to isolate different cluster nodes.
```

**Comment by disksing** [comment]
😢 The content is copied from [command-line-flags document](https://github.com/pingcap/docs/blob/master/command-line-flags-for-pd-configuration.md). The inconsistency is inherited from the inconsistency of that document.

File: pd-configuration-file.md [hunk 6]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
+### `client-urls`
+
+- The list of client URLs to be listened to for client traffic
+- Default value: `"http://127.0.0.1:2379"`
+- To deploy a cluster, you must specify the IP address of the current host as `client-urls` (for example, `"http://192.168.100.113:2379"`). If the cluster runs on Docker, specify the IP address of Docker as `"http://0.0.0.0:2379"`.
+
+### `advertise-client-urls`
+
+- The list of advertise URLs for client traffic from the outside
+- Default value: `"${peer-urls}"`
```
Comment (TomShawn) [inline pd-configuration-file.md:35]:
```suggestion
- Default value: `"${client-urls}"`
```

File: pd-configuration-file.md [hunk 7]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
+- In some situations such as the Docker or NAT network environment, if a client cannot connect to PD through the default client URLs to be listened to by PD, you must manually set the advertise client URLs.
```
Comment (TomShawn) [inline pd-configuration-file.md:36]:
```suggestion
- In some situations such as in the Docker or NAT network environment, if a client cannot access PD through the default client URLs listened to by PD, you must manually set the advertise client URLs.
```

File: pd-configuration-file.md [hunk 8]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
+- To start multiply PD nodes, use a unique name for each node.
+
+### `data-dir`
+
+- The directory in which PD stores data
+- Default value: `default.${name}"`
+
+### `client-urls`
+
+- The list of client URLs to be listened to for client traffic
```
Comment (TomShawn) [inline pd-configuration-file.md:28]:
```suggestion
- The list of client URLs to be listened to by PD
```

File: pd-configuration-file.md [hunk 9]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
+- Default value: `"http://127.0.0.1:2379"`
+- To deploy a cluster, you must specify the IP address of the current host as `client-urls` (for example, `"http://192.168.100.113:2379"`). If the cluster runs on Docker, specify the IP address of Docker as `"http://0.0.0.0:2379"`.
+
+### `advertise-client-urls`
+
+- The list of advertise URLs for client traffic from the outside
```
Comment (TomShawn) [inline pd-configuration-file.md:34]:
```suggestion
- The list of advertise URLs for the client to access PD
```

File: pd-configuration-file.md [hunk 10]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
+- Default value: `"${peer-urls}"`
+- In some situations such as the Docker or NAT network environment, if a client cannot connect to PD through the default client URLs to be listened to by PD, you must manually set the advertise client URLs.
+- For example, the internal IP address of Docker is `172.17.0.1`, while the IP address of the host is `192.168.100.113` and the port mapping is set to `-p 2380:2380`. In this case, you can set `advertise-client-urls` to `"http://192.168.100.113:2380"`. The client can find this service through `"http://192.168.100.113:2380"`.
+
+### `peer-urls`
+
+- The list of peer URLs to be listened to for peer traffic
```
Comment (TomShawn) [inline pd-configuration-file.md:41]:
```suggestion
- The list of peer URLs to be listened to by a PD node
```

File: pd-configuration-file.md [hunk 11]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
+- Default value: `"http://127.0.0.1:2380"`
+- To deploy a cluster, you must specify `peer-urls` as the IP address of the current host, such as `"http://192.168.100.113:2380"`. If the cluster runs on Docker, specify the IP address of Docker as `"http://0.0.0.0:2380"`.
+
+### `advertise-peer-urls`
+
+- The list of advertise URLs for peer traffic from the outside
```
Comment (TomShawn) [inline pd-configuration-file.md:47]:
```suggestion
- The list of advertise URLs for other PD nodes (peers) to access a PD node
```

File: pd-configuration-file.md [hunk 12]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
+- Default: `"${peer-urls}"`
+- In some situations such as the Docker or NAT network environment, if the other nodes cannot connect to PD through the default peer URLs to be listened to by PD, you must manually set the advertise peer URLs.
```
Comment (TomShawn) [inline pd-configuration-file.md:49]:
```suggestion
- In some situations such as in the Docker or NAT network environment, if the other nodes (peers) cannot access the PD node through the default peer URLs listened to by this PD node, you must manually set the advertise peer URLs.
```

File: pd-configuration-file.md [hunk 13]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
+- The list of advertise URLs for client traffic from the outside
+- Default value: `"${peer-urls}"`
+- In some situations such as the Docker or NAT network environment, if a client cannot connect to PD through the default client URLs to be listened to by PD, you must manually set the advertise client URLs.
+- For example, the internal IP address of Docker is `172.17.0.1`, while the IP address of the host is `192.168.100.113` and the port mapping is set to `-p 2380:2380`. In this case, you can set `advertise-client-urls` to `"http://192.168.100.113:2380"`. The client can find this service through `"http://192.168.100.113:2380"`.
+
+### `peer-urls`
+
+- The list of peer URLs to be listened to for peer traffic
+- Default value: `"http://127.0.0.1:2380"`
+- To deploy a cluster, you must specify `peer-urls` as the IP address of the current host, such as `"http://192.168.100.113:2380"`. If the cluster runs on Docker, specify the IP address of Docker as `"http://0.0.0.0:2380"`.
```
Comment (TomShawn) [inline pd-configuration-file.md:43]:
```suggestion
- When you deploy a cluster, you must specify `peer-urls` as the IP address of the current host, such as `"http://192.168.100.113:2380"`. If the cluster runs on Docker, specify the IP address of Docker as `"http://0.0.0.0:2380"`.
```

File: pd-configuration-file.md [hunk 14]
```diff
@@ -12,21 +12,76 @@ The PD configuration file supports more options than command-line parameters. Yo
...
+### `data-dir`
+
+- The directory in which PD stores data
+- Default value: `default.${name}"`
+
+### `client-urls`
+
+- The list of client URLs to be listened to for client traffic
+- Default value: `"http://127.0.0.1:2379"`
+- To deploy a cluster, you must specify the IP address of the current host as `client-urls` (for example, `"http://192.168.100.113:2379"`). If the cluster runs on Docker, specify the IP address of Docker as `"http://0.0.0.0:2379"`.
```
Comment (TomShawn) [inline pd-configuration-file.md:30]:
```suggestion
- When you deploy a cluster, you must specify the IP address of the current host as `client-urls` (for example, `"http://192.168.100.113:2379"`). If the cluster runs on Docker, specify the IP address of Docker as `"http://0.0.0.0:2379"`.
```

**Pull request merged**